README.md

```diff
@@ -0,0 +1,119 @@
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Budget-App <a name="about-project"></a>
+
+**Budget-App** is a full-stack web application developed using Ruby on Rails. It provides user authentication with login and logout functionality, allowing users to securely access their data. With Budget-App, users can add categories to organize their expenses and incomes, and then add transactions for each category to keep track of their spending. The application also features a modern and visually appealing design that enhances the user experience. The design was created by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), a highly acclaimed designer whose portfolio is showcased on Behance.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Developed using Ruby on Rails.
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Register and login features**
+- **Database where you can add categories and transactions**
+- **Implemented design from the sneak peek wireframe**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://budget-app-qrxi.onrender.com)
+
```

Comment:
- [ ] `Required`
- [ ] Good readme file
- [ ] Good job on the live link 👍 
- [ ] You are also required to add a `Video presentation`  
1. Please recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video should contain a demo of the project, highlighting all the project features.
5. The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Thanks for your review @mwanawabangona , I forgot to add it to the `README.md` file